src/main/java/at/hannibal2/skyhanni/config/commands/Commands.kt

```diff
@@ -414,6 +415,9 @@ object Commands {
         registerCommand("pt", "Transfer the party to another party member") { PartyCommands.transfer(it) }
         registerCommand("pp", "Promote a specific party member") { PartyCommands.promote(it) }
         registerCommand("pd", "Disbands the party") { PartyCommands.disband() }
+
+        registerCommand("gfs", "Get from Sack (queued)") { GetFromSackData.commandHandler(it) }
```

Comment:
This suggestion broke everything. Because when the MessageSendToServerEvent is used it becomes impossible to send the command to the server. Because it recursively calls it self and block it self.
Eg. Only typed the command once
![image](https://github.com/hannibal002/SkyHanni/assets/85900443/dad11f51-600f-4afe-a16b-52f084f313dc)

Reply:
Find a way to differentiate between commands sent from users and from mods.
i think we have a way to do this already. when im wrong, just keep the current command as a workaround ig.

Reply:
implemented